pkg/binary/envoy/debug/node.go

```diff
@@ -65,34 +63,25 @@ func (n *nodeCollection) ps() error {
 	defer cancel()
 
 	processes, err := process.ProcessesWithContext(ctx)
-	if ctx.Err() == context.DeadlineExceeded {
-		log.Debugf("timeout getting a list of running processes")
-		return nil
+	if err == nil {
+		err = ctx.Err()
 	}
 	if err != nil {
-		log.Debugf("unable to get list of running processes: %v", err)
-		return nil
+		if err.Error() == "not implemented yet" { // internal error used by gopsutil
+			return nil // It will never work, so there's no reason to bother users
+		}
+		return fmt.Errorf("unable to get list of running processes: %w", err)
 	}
 
-	return printProcessTable(ctx, f, processes)
-}
-
-func printProcessTable(ctx context.Context, out io.Writer, processes []*process.Process) error {
-	w := tabwriter.NewWriter(out, 0, 8, 5, ' ', 0)
-	fmt.Fprintln(w, "PID\tUSERNAME\tSTATUS\tRSS\tVSZ\tMINFLT\tMAJFLT\tPCPU\tPMEM\tARGS")
-	for _, p := range processes {
-		proc := safeProc(ctx, p)
-		if proc == empty { // Ignore, but continue. The process could have died between the process listing and now.
-			continue
+	parsed, err := parseProcessTable(ctx, processes)
+	if len(parsed) == 0 {
+		if err != nil {
+			return fmt.Errorf("unable to parse any processes: %w", err)
```

Comment:
only looking at the error when we parsed nothing helps us allow partial process table listing (ex columns unsupported to be blank)